conductr_cli/bundle_core_info.py

```diff
@@ -1,22 +1,44 @@
+import datetime
+
 from conductr_cli.conduct_info_inspect import has_bundle_id, has_bundle_name
 
 
 class BundleCoreInfo(object):
-    def __init__(self, bundle_id, bundle_name, bundle_digest, configuration_digest):
+    def __init__(self, bundle_id, bundle_name, bundle_digest, configuration_digest,
+                 scale=0, start_time=datetime.datetime.max, compatibility_version=None):
         self.bundle_id = bundle_id
         self.bundle_name = bundle_name
         self.bundle_digest = bundle_digest
         self.configuration_digest = configuration_digest
         self.bundle_name_with_digest = '{0}-{1}'.format(bundle_name, bundle_digest)
         self.bundle_name_with_configuration_digest = '{0}-{1}'.format(bundle_name, configuration_digest)
+        self.scale = scale
+        self.start_time = start_time
+        self.compatibility_Version = compatibility_version
 
     @classmethod
     def from_bundles(cls, bundles_json):
+        def get_start_time(bundle):
+            if bundle.get('bundleExecutions') is None:
+                return datetime.datetime.max
+            if len(bundle['bundleExecutions']) > 0:
+                start = (bundle['bundleExecutions'])[0].get('startTime', None)
+                if start is not None:
+                    return datetime.datetime.strptime(start.split('.')[0], '%Y-%m-%dT%H:%M:%S')
+                else:
+                    return datetime.datetime.max
+            else:
+                return datetime.datetime.max
```

Comment:
Just a suggestion: `get_start_time` looks functionally correct. I think it could be easier to follow if you just return in two places -- once at the very inner nest, and then once at the end of the function.

ie.
```
def get_start_time(bundle):
  if bundle.get(...) is not None and len(...) > 0:
    start = ...
    if start is not None:
       return ...

  return default value